castle/test/commands/authenticate_test.py

```diff
@@ -1,89 +1,85 @@
-from castle.test import unittest
+from castle.test import mock, unittest
 from castle.command import Command
 from castle.commands.authenticate import CommandsAuthenticate
-from castle.context.merger import ContextMerger
 from castle.exceptions import InvalidParametersError
+from castle.utils import clone
 
 
-def default_payload():
+def default_options():
+    """Default options include all required fields."""
     return {'event': '$login.authenticate', 'user_id': '1234'}
 
+
+def default_options_plus(**extra):
+    """Default options plus the given extra fields."""
+    options = default_options()
+    options.update(extra)
+    return options
+
+
+def default_command_with_data(**data):
+    """What we expect the authenticate command to look like."""
+    return Command(
+        method='post',
+        path='authenticate',
+        data=dict(sent_at=mock.sentinel.timestamp, **data)
+    )
+
+
 class CommandsAuthenticateTestCase(unittest.TestCase):
+
+    def setUp(self):
+        # patch timestamp to return a known value
+        timestamp_patcher = mock.patch('castle.commands.authenticate.timestamp')
+        self.mock_timestamp = timestamp_patcher.start()
+        self.mock_timestamp.return_value = mock.sentinel.timestamp
+        self.addCleanup(timestamp_patcher.stop)
+
     def test_init(self):
-        self.assertIsInstance(CommandsAuthenticate({}).context_merger, ContextMerger)
+        obj = CommandsAuthenticate({})
+        self.assertEqual(obj.context, {})
```

Comment:
I think this would be a more effective test if you used a sentinel:
```python
context = mock.sentinel.test_init_context
obj = CommandsAuthenticate(context)
self.assertEqual(obj.context, context)
```